mautic/mautic
# Maut 3399 do not sell list

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | Y
| Automated tests included? | Y
| Related user documentation PR URL | 
| Related developer documentation PR URL | https://dev.maxmind.com/geoip/privacy-exclusions-api/
| Issues addressed (#s or URLs) | 
| BC breaks? | MaxMind auth information must be in the form: ID:LicenseKey
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description: MaxMind provides and API endpoint to fetch the list of IP's that should not be tracked due to CCPA requests. A command will exist in the cloud agent that is able to be executed to fetch the updated list of IP's and write it to the shared filesystem.

#### Steps to test this PR:
0. Load up [this PR](https://mautibox.com)
1. Add to parameters the path to file: 	'maxmind_do_not_sell_list_path' => './maxmind.json',
2. Run  app/console mautic:donotsell:download
3. Check the JSON file in path

#### List backwards compatibility breaks:
1. MaxMind auth information must be in the form: ID:LicenseKey
